Cargo.toml

```diff
@@ -42,5 +42,8 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 thiserror = "1.0"
 uint = "0.8"
-web3 = "0.11"
+web3 = "0.12"
 zeroize = "1.1"
+
+[dev-dependencies]
+tokio = { version = "0.2", features = ["macros"] }
```

Comment:
The web3 provided http transport only runs on the Tokio runtime so for our examples we need this.